__test__/UserCollectableAttributesWithMockedDefinitions.test.js

```diff
@@ -58,6 +58,32 @@ describe('UCA Constructions tests', () => {
     expect(uca).toBeDefined();
   });
 
+  test('Should construct UCA with array', () => {
+    const identifier = 'cvc:Collection.records';
+    const value = [
+      'Belo Horizonte',
+      'Brazil',
+      'Minas Gerais',
+    ];
+    const uca = new UCA(identifier, value);
+    expect(uca).toBeDefined();
+    expect(uca.getPlainValue()).toBeDefined();
```

Comment:
shouldn't we check the actual value to make sure it's correct, rather than just checking that is exists?